data/hi/retired/Val-Okimoto-0893bbd0-5e0d-4dc4-aca4-cb771597301b.yml

```diff
@@ -7,7 +7,7 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/okimoto.jpg
 party:
 - name: Republican
 roles:
-- end_date: '2022-12-30'
+- end_date: '2022-11-07'
```

Comment:
She assumed office on 1/2/2023, she changed from republican to non-partisan during the recent election.